ext/opcache/zend_accelerator_module.c

```diff
@@ -491,6 +494,26 @@ void zend_accel_info(ZEND_MODULE_INFO_FUNC_ARGS)
 			php_info_print_table_row(2, "Hash keys restarts", buf);
 			snprintf(buf, sizeof(buf), ZEND_ULONG_FMT, ZCSG(manual_restarts));
 			php_info_print_table_row(2, "Manual restarts", buf);
+
+			start_time = php_format_date("Y-m-d H:i:s e", sizeof("Y-m-d H:i:s e")-1, ZCSG(start_time), 1);
```

Comment:
`DATE_FORMAT_ISO8601` might fit perhaps.

Reply:
Well, we can obtain the value of the constant via `zend_get_constant_str("DATE_ISO8601", ...);` or just hardcode the format as it is now. 

Which approach would you prefer?

It looks like c-constants `DATE_FORMAT_ISO8601` and `DATE_FORMAT_ISO8601_LARGE_YEAR` are defined in `php_date.c` and aren't accessible here.

Reply:
preferably let s keep them private as they are.

Reply:
changed to `zend_get_constant_str("DATE_ISO8601")`